3. WebApp calls several APIS (incremental consent and CA)/Controllers/HomeController.cs

```diff
@@ -56,7 +56,7 @@ public async Task<IActionResult> Profile()
         public async Task<IActionResult> Tenants()
         {
             var accessToken =
-                await tokenAcquisition.GetAccessTokenOnBehalfOfUser(HttpContext, new[] { $"{ArmApiOperationService.ArmResource}/user_impersonation" });
+                await tokenAcquisition.GetAccessTokenOnBehalfOfUser(HttpContext, new[] { $"{ArmApiOperationService.ArmResource}user_impersonation" });
 
             var tenantIds = await armOperations.EnumerateTenantsIdsAccessibleByUser(accessToken);
 /*
```

Comment:
Everything works , except the following lines in this controller

            Uri blobUri = new Uri("https://blobstorageazuread.blob.core.windows.net/sample-container/Blob1.txt");
            CloudBlockBlob blob = new CloudBlockBlob(blobUri, storageCredentials);
await blob.UploadTextAsync("Blob created by Azure AD authenticated user.");

I'll investigate more, but the instructed provided here and in the preceding did not had any steps to configure Azure storage. Or did I miss anything

Reply:
@kalyankrishna1 : this is "normal". I have not added the readme yet for this one. and there is work in progress with someone from CSE updating a Storage article. this will come.